### PR TITLE
Remove TODO from VS.gitignore

### PR DESCRIPTION
**Pull request by jamesqo**
**Reasons for making this change:**

I search my GitHub repo regularly for the string "TODO" and this always shows up. Please consider removing the todo; I'm also open to changing it to something else like `Note: `.

### CODE REVIEW

**Comment by shiftkey** [comment]
@jamesqo good point!

File: VisualStudio.gitignore [hunk 1]
```diff
@@ -151,7 +151,7 @@ publish/
 # Publish Web Output
 *.[Pp]ublish.xml
 *.azurePubxml
-# TODO: Comment the next line if you want to checkin your web deploy settings
```
Comment (shiftkey) [inline VisualStudio.gitignore]: :thumbsup: to some other prefix here. I'm not really sure if there's a better way here to indicate that you _can_ introduce an optional rule here, but `Note:` is a fine substitute.

Comment (jamesqo) [inline VisualStudio.gitignore]: OK, I went with `Note: `.

**Comment by shiftkey** [comment]
@jamesqo thanks!

**Pull request merged**